src/openms/include/OpenMS/KERNEL/MSExperiment.h

```diff
@@ -538,6 +538,15 @@ namespace OpenMS
     **/
     const MSChromatogram getTIC(float rt_bin_size=0) const;
 
+		/// returns a const reference to the description of the applied data processing
```

Comment:
the DataProcessing for MSExperiment is a bit confusing. Its defined via the spectra, i.e.
`exp[s].getDataProcessing()`. Follow `https://github.com/cbielow/OpenMS/blob/develop/src/topp/InternalCalibration.cpp#L371` to see how its added and how to check it has been done (--> checking the first spectrum's DPs should be enough)

Reply:
i.e. you can remove all the DataProcessing members/accessors you added :/